nexus-sdk-js/src/utils/http.ts

```diff
@@ -131,6 +136,30 @@ export function httpPut(
     });
 }
 
+export function httpPostFile(
```

Comment:
Is it necessary to create a new function? `httpPost` was designed to be generic enough. You could do something like `httpPost<File>(url, file, {extraHeaders: { 'Content-Type': 'multipart/form-data;' })`

You can add another generic for the response type if needed `httpPost<T = object, U = any>(url: string, body: T): Promise<U>`

Reply:
For sure, I'll make it more generic, was just testing some concepts out